pyserini/encoded_query_info.py

```diff
@@ -9,24 +9,74 @@
         "total_queries": 6980,
         "downloaded": False
     },
-    "nq-dev-dpr": {
-        "description": "Natural Question dev set questions encoded by DPR question encoder",
+    "nq-dev-dpr-multi": {
+        "description": "Natural Question dev set questions encoded by DPR question encoder trained on multiset",
         "urls": [
-            "https://www.dropbox.com/s/xbhtpib9irkek94/query-embedding-nq-dev-dpr-20210121-a5983b.tar.gz?dl=1",
+            "https://www.dropbox.com/s/tit9qn0oxmerlyk/query-embedding-nq-dev-dpr-multi-20210129-3a276c.tar.gz?dl=1",
         ],
-        "md5": "cc4168964b4aac0fdc2b071d1c54c8ce",
-        "size (bytes)": 25115164,
+        "md5": "c584f779c56d6396d3aab5a1f4be53f0",
+        "size (bytes)": 25121360,
         "total_queries": 8757,
         "downloaded": False
     },
-    "nq-test-dpr": {
-        "description": "Natural Question test set questions encoded by DPR question encoder",
+    "nq-test-dpr-multi": {
+        "description": "Natural Question test set questions encoded by DPR question encoder trained on multiset",
         "urls": [
-            "https://www.dropbox.com/s/q69lxg1irier7sw/query-embedding-nq-test-dpr-20210121-a5983b.tar.gz?dl=1",
+            "https://www.dropbox.com/s/hxvnhe6evk0vdeu/query-embedding-nq-test-dpr-multi-20210129-3a276c.tar.gz?dl=1",
         ],
-        "md5": "f91de0a295db2907a47cfb00961e98e6",
-        "size (bytes)": 10357275,
+        "md5": "dd92355c7661cf23a802e793cd5a73ff",
+        "size (bytes)": 10359974,
         "total_queries": 3610,
         "downloaded": False
+    },
+    "trivia_qa-dev-dpr-multi": {
```

Comment:
my suggestion would be to put `dpr` first, so if we do an `ls`, the `dpr` datasets are grouped together.

Reply:
sure

Reply:
since there are many naming rules that need to discuss/change, I'll do that in a separate PR. Maybe together with https://github.com/castorini/pyserini/issues/334

Reply:
sure